src/appmixer/intelswift/core/HookUpdateContactHubspot/HookUpdateContactHubspot.js

```
@@ -0,0 +1,26 @@
+module.exports = {
+
+    async receive(context) {
+
+        const {  syncId, ...meta } = context.messages.in.content;
+
+        // remaining to handles tenantId and projectId
+
+        let body = {};
+
+        body.hubspotId = syncId;
+        
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)